admin/client/src/state/records/RecordsReducer.js

```diff
@@ -7,8 +7,12 @@ function recordsReducer(state = initialState, action) {
   let records = null;
   let recordType = null;
   let record = null;
+  let apolloRecordRelation = {};
 
   switch (action.type) {
+    case 'APOLLO_QUERY_RESULT':
```

Comment:
I think this leaked from the GraphQL changes, don't think this belongs in this PR :D